src/schema/v2/user.ts

```diff
@@ -96,260 +96,303 @@ export const ProfileAccessField: GraphQLFieldConfig<any, ResolverContext> = {
-      type: GraphQLString,
-      resolve: ({ paddle_number }) => paddle_number,
-    },
-    partnerAccess: PartnerAccessField,
-    profileAccess: ProfileAccessField,
-    receivePurchaseNotification: {
-      description: "This user should receive purchase notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_purchase_notification }) =>
-        receive_purchase_notification,
-    },
-    receiveOutbidNotification: {
-      description: "This user should receive outbid notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_outbid_notification }) => receive_outbid_notification,
-    },
-    receiveLotOpeningSoonNotification: {
-      description: "This user should receive lot opening notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_lot_opening_soon_notification }) =>
-        receive_lot_opening_soon_notification,
-    },
-    receiveSaleOpeningClosingNotification: {
-      description:
-        "This user should receive sale opening/closing notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_sale_opening_closing_notification }) =>
-        receive_sale_opening_closing_notification,
-    },
-    receiveNewWorksNotification: {
-      description: "This user should receive new works notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_new_works_notification }) =>
-        receive_new_works_notification,
-    },
-    receiveNewSalesNotification: {
-      description: "This user should receive new sales notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_new_sales_notification }) =>
-        receive_new_sales_notification,
-    },
-    receivePromotionNotification: {
-      description: "This user should receive promotional notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_promotion_notification }) =>
-        receive_promotion_notification,
-    },
-    receiveOrderNotification: {
-      description: "This user should receive order notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_order_notification }) => receive_order_notification,
-    },
-    receiveViewingRoomNotification: {
-      description: "This user should receive viewing room notifications",
-      type: GraphQLBoolean,
-      resolve: ({ receive_viewing_room_notification }) =>
-        receive_viewing_room_notification,
-    },
-    savedArtworksConnection: {
-      type: artworkConnection.connectionType,
-      args: pageable({}),
-      resolve: async ({ id }, args, { savedArtworksLoader }) => {
-        if (!savedArtworksLoader) return null
+        }),
+        resolve: (result) => result,
+      },
+      paddleNumber: {
+        description: "The paddle number of the user",
+        type: GraphQLString,
+        resolve: ({ paddle_number }) => paddle_number,
+      },
+      partnerAccess: PartnerAccessField,
+      profileAccess: ProfileAccessField,
+      purchasesConnection: {
```

Comment:
Instead of `purchasesConnection`, what do you think of `purchasedArtworksConnection`? It's a bit more verbose, but I think it's a bit more accurate, as it's a connection of artworks (on the node), and we generally have "artworks connection" in the name of fields whose type is some connection of artworks.

#minoir #non-blocker

Reply:
done.

I've tried converting this [require](https://github.com/artsy/metaphysics/pull/4370/files#diff-53a08671c7ab0c03c56b68075478ab8b4b79e04a1c3f946e29aed49569a66abcR100) into an `import` statement but then the app breaks on merging schemas. It isn't apparent to me why this doesn't work with `import`. Seems very similar to how `interestsConnection` is imported and used.

```
[V2] Error merging schemas: TypeError: Cannot read property 'name' of undefined
...
```

Perhaps something to do with the fact that this is an [any](https://github.com/artsy/metaphysics/pull/4370/files#diff-1a006ac61a6f2591afd29633a086ff02c200772a65db65c54a4a2ec8a782f5c7R13) type?

Reply:
@ovasdi - its worth noting that there's no relationship whatsoever between javascript runtime errors and typescript errors. Meaning: when we add type annotations to our code, this is a compile time layer that is stripped out by [babel](https://babeljs.io/) before code is actually served by an app to execute at runtime. 

[See here where we configure it](https://github.com/artsy/metaphysics/blob/main/babel.config.js#L11).

Reply:
@damassi thanks for the explanation.